packages/fxa-settings/src/models/contexts/NimbusContext.ts

```diff
@@ -160,7 +168,9 @@ export function NimbusProvider({ children }: NimbusProviderProps) {
     config?.nimbus.enabled,
     config?.nimbus.preview,
     uniqueUserId,
-    currentLocale
+    currentLocale,
+    currentAccountUid,
+    accountMetricsEnabled,
   ]);
 
   const value: NimbusContextValue = useMemo(() => ({
```

Comment:
Hm, when I navigate from email-first to sign-in after a fresh `/clear`, I get this error in the console:

<img width="350" height="451" alt="Image" src="https://github.com/user-attachments/assets/e85529b4-3412-486b-8a7f-c63f26ab2fa8" />

I wouldn't expect this to rerender at this point, given local storage hasn't changed yet? Or maybe it did change due to Glean entry stuff and we only need to rerender on `currentAccountUid` and not `accounts` if that changes there?

I don't see the error again at least when I enter the account password and the `useEffect` correctly runs, it seems to work, which is neat!

Reply:
@LZoog I havent been able to reproduce this :/